components/components.css

```diff
@@ -1 +1,22 @@
-@import './alert.css'
\ No newline at end of file
+@import './alert.css';
+
+.btn-close {
+    box-sizing: content-box;
+    width: 1rem;
```

Comment:
Why you have put box-sizing content box??

Reply:
This must have been an oversight. Fixed that.